nilearn/decoding/space_net.py

```diff
@@ -835,11 +823,23 @@ def _set_coef_and_intercept(self, w):
         if self.w_.ndim == 1:
             self.w_ = self.w_[np.newaxis, :]
         self.coef_ = self.w_[:, :-1]
-        if self.is_classif:
+        if self._is_classification:
             self.intercept_ = self.w_[:, -1]
         else:
             self._set_intercept(self.Xmean_, self.ymean_, self.Xstd_)
 
+    def _return_loss_value(self):
+        """Set loss value for instances where it is not defined.
+
+        For SpaceNetRegressor it is always "mse".
+        """
+        loss = getattr(self, "loss", None)
+        if loss is None:
+            loss = "logistic"
+            if not self._is_classification:
+                loss = "mse"
```

Comment:
The same here.

`SpaceNetRegressor` can have `_return_loss_value` function that returns `mse` and `SpaceNetRegressor` should have `_return_loss_value` function that implements desired behavior.